lang/en/lang.php

```diff
@@ -213,6 +213,8 @@
         'property_datepicker_year_range' => 'Year range',
         'property_datepicker_year_range_description' => 'Number of years either side (eg 10) or array of upper/lower range (eg [1900,2015]). Leave empty for the default value (10).',
         'property_datepicker_year_range_invalid_format' => 'Invalid year range format. Use number (eg "10") or array of upper/lower range (eg "[1900,2015]")',
+        'property_datepicker_format' => 'Format',
+        'property_datepicker_year_format_description' => 'Setup custom format date. Default format is YYYY-MM-DD',
```

Comment:
@host138 Would you mind adjusting the grammar of this for us?
```suggestion
        'property_datepicker_year_format_description' => 'Define a custom date format for storage. The default format is YYYY-MM-DD',
```

Reply:
Note too that the date picker widget uses different formatting to the PHP `date` function - so the default format is actually "Y-m-d"